### ISSUE TITLE
Uncaught TypeError when "tabbing" over volume controls

### ISSUE THREAD

**Issue by ChristiaanScheermeijer** [jwplayer/jwplayer#2766]
### Expected Behavior

When using the tab button to navigate over the volume button, it should show the volume slider. After leaving the volume button, by pressing tab or shift+tab, it should close the volume slider.

### Actual Behavior

The volume slider stays visible and a TypeError is thrown:

```
Uncaught TypeError: Failed to execute 'elementFromPoint' on 'Document': The provided double value is non-finite.
    at HTMLDivElement.f
```

### Steps to reproduce

1. Press tab a few times until the volume slider shows
2. Press tab again

### Environment

- macOS | High Sierra 10.13.3 
- Google Chrome | 64.0.3282.186
- JW Player 8.1.11

----------------------------------------------------------------------------------------

This template is provided as a guide to filing bugs and feature requests. Please replace 
all text *in italics* with details describing your issue. When submitting a feature request, 
"Steps to reproduce" can be substituted with a user story or list of acceptance criteria.

**Comment by hghazzi** [comment]
@ChristiaanScheermeijer This will be resolved in `v8.2.0`

**Comment by hghazzi** [comment]
Fixed by https://github.com/jwplayer/jwplayer/pull/2735
